Added UVOstat logo to landing

PeterPsota opened the pull request:
![image](https://user-images.githubusercontent.com/18385255/45755845-b6b31d80-bc1f-11e8-951a-aa5f74cbb929.png)

mirecmrozek commented:
Suhlas, je to takto trochu na kope. Malo by stacit pridat classname `ml-3` na riadok 115. @PeterPsota pridaj to prosim tiez do tohto PR.

PeterPsota commented:
Fixed the image and added margin.
Didn't use ml-3 as it worsened alignment on mobile.

![ml-3](https://user-images.githubusercontent.com/18385255/45817498-48359480-bcdf-11e8-9fb2-05c9ef82f4d7.png)
^ml-3
![image](https://user-images.githubusercontent.com/18385255/45817483-3f44c300-bcdf-11e8-852c-a79aecb4df72.png)
